server/src/test/java/org/opensearch/search/profile/query/ConcurrentQueryProfileBreakdownTests.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.search.profile.query;
+
+import org.opensearch.test.OpenSearchTestCase;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ConcurrentQueryProfileBreakdownTests extends OpenSearchTestCase {
+
+    public void testBuildQueryProfileBreakdownMap() {
+        Map<String, Long> testMap = new HashMap<>();
+        testMap.put("set_min_competitive_score_count", 0L);
+        testMap.put("match_count", 0L);
+        testMap.put("score_start_time", 0L);
+        testMap.put("shallow_advance_count", 0L);
+        testMap.put("create_weight_start_time", 1598347679188617L);
```

Comment:
Can you define a constant time and define these numbers using that as a base?  For example, say:
START_TIME = X
testMap.put("create_weight_start_time", START_TIME);
testMap.put("score_count", START_TIME + 2);

Reply:
Hi @kkmr, these numbers differ from case to case. I don't think it's needed as a base constant here. Instead, I have added all the query timing types as constants.